hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -1088,72 +828,95 @@ public void testEntrySet_withPredicate() {
+        assertTrue("get latency", 0 < stats.get("getLatency"));
+        assertTrue("remove latency", 0 < stats.get("removeLatency"));
+    }
+
+    private void addLocalMapStats(HashMap<String, Long> stats, LocalMapStats serverMapStats) {
+        addValueToHashMap(stats, "put", serverMapStats.getPutOperationCount());
+        addValueToHashMap(stats, "get", serverMapStats.getGetOperationCount());
+        addValueToHashMap(stats, "remove", serverMapStats.getRemoveOperationCount());
+        addValueToHashMap(stats, "putLatency", serverMapStats.getTotalPutLatency());
+        addValueToHashMap(stats, "getLatency", serverMapStats.getTotalGetLatency());
+        addValueToHashMap(stats, "removeLatency", serverMapStats.getTotalRemoveLatency());
+    }
+
+    private void addValueToHashMap(HashMap<String, Long> stats, String key, Long value) {
+        Long oldValue = stats.get(key);
+        stats.put(key, (oldValue == null) ? value : oldValue + value);
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testAddLocalEntryListener(){
-        final IMap map = client.getMap(randomString());
-        map.addLocalEntryListener(new DumEntryListener());
+    public void testAddLocalEntryListener() {
+        randomStringMap.addLocalEntryListener(new DummyEntryListener());
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testAddLocalEntryListener_WithPredicate(){
-        final IMap map = client.getMap(randomString());
-        map.addLocalEntryListener(new DumEntryListener(), new DumPredicate(), true);
+    public void testAddLocalEntryListener_WithPredicate() {
+        randomStringMap.addLocalEntryListener(new DummyEntryListener(), new DummyPredicate(), true);
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testAddLocalEntryListener_WithPredicateAndKey(){
-        final IMap map = client.getMap(randomString());
-        map.addLocalEntryListener(new DumEntryListener(), new DumPredicate(), "Key", true);
+    public void testAddLocalEntryListener_WithPredicateAndKey() {
+        randomStringMap.addLocalEntryListener(new DummyEntryListener(), new DummyPredicate(), KEY, true);
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testLocalKeySet(){
-        final IMap map = client.getMap(randomString());
-        map.localKeySet();
+    public void testLocalKeySet() {
+        randomStringMap.localKeySet();
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testLocalKeySet_WithPredicate(){
-        final IMap map = client.getMap(randomString());
-        map.localKeySet(new DumPredicate());
+    public void testLocalKeySet_WithPredicate() {
+        randomStringMap.localKeySet(new DummyPredicate());
     }
 
-    static class DumEntryListener implements EntryListener {
+    private static class DummyEntryListener implements EntryListener {
```

Comment:
Why don't you create a mock using Mockito of you totally don't care about its content?

EntryListener entryListener = mock(EntryListener.class);